src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionService.java

```diff
@@ -38,6 +40,8 @@
 
     private static final String OBJECTION_NOT_FOUND_MESSAGE = "Objection with id: %s, not found";
     private static final String ATTACHMENT_NOT_FOUND_MESSAGE = "Attachment with id: %s, not found";
+    private static final String DELETE_ERROR_MESSAGE = "Unable to delete attachment %s, status code %s";
```

Comment:
Again, just some context in the name of these two constants - so prefixed with 'ATTACHMENT' and maybe remove 'ERROR' since the constants already defined above don't have this.